gradle/libs.versions.toml

```diff
@@ -31,7 +33,7 @@ firebase-bom = "com.google.firebase:firebase-bom:32.6.0"
 firebase-analytics = { module = "com.google.firebase:firebase-analytics-ktx" }
 firebase-crashlytics = { module = "com.google.firebase:firebase-crashlytics-ktx" }
 firebase-storage = { module = "com.google.firebase:firebase-storage-ktx" }
-firebase-firestore = { module = "com.google.firebase:firebase-firestore" }
+firebase-firestore = { module = "dev.gitlive:firebase-firestore", version.ref = "firebase" }
```

Comment:
It's listed on `firebaseopensource` so it seems to be the best option at this moment. 
https://firebaseopensource.com/projects/gitliveapp/firebase-kotlin-sdk/ 
Github: https://github.com/gitliveapp/firebase-kotlin-sdk/

Reply:
Seems legit, I would keep it simply `firebase`